src/apps/lwaftr/ndp.lua

```diff
@@ -151,6 +151,12 @@ function is_neighbor_solicitation_for_ips(pkt, local_ips)
    return false
 end
 
+local function to_ether_addr(pkt, offset)
+   local ether_src = ffi.new("uint8_t[?]", 6)
+   ffi.copy(ether_src, pkt.data + offset, 6)
```

Comment:
Most of this file just implicitly uses pointers rather than explicit copies. Given the scope this is used from, the packet will still be live, so I don't see any reason to create garbage with an ffi.copy.